### PR TITLE
Update ISSUE_TEMPLATE.md - added code blocks

### LINKED ISSUE #3952: Meta issue: add code blocks to the relevant sections of the new-issue template to facilitate formatting
Not sure where to post this, so I'm doing it here.

Below is a modified version of the new-issue template (use the markdown source) that adds fenced code blocks / output blocks  to the `Steps to reproduce`, `Expected behavior` and `Actual behavior` sections, to facilitate proper formatting.

---

Steps to reproduce
------------------

```powershell

```

Expected behavior
-----------------

```none

```

Actual behavior
---------------

```none

```

Environment data
----------------

```powershell
> $PSVersionTable

```

### PR DESCRIPTION
**Pull request by mklement0**
Fix #3952. Adds fenced code blocks / output blocks to the `Steps to reproduce`, `Expected behavior` and `Actual behavior` sections, so as to facilitate proper formatting.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -14,14 +14,23 @@ If it's not a bug, please remove the template and elaborate the issue in your ow
...
 ------------------
 
+```powershell
+
+```
 
 Expected behavior
 -----------------
 
+```none
```
Comment (SteveL-MSFT) [inline .github/ISSUE_TEMPLATE.md]: I would suggest defaulting the code block to `powershell` instead of `none`, same below

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (mklement0) [inline .github/ISSUE_TEMPLATE.md]: @SteveL-MSFT: `powershell` is for highlighting PowerShell _source code_, whereas  _output_ will generally _not_ be _source code_ - in which case highlighting as if it _were_ can be distracting.

Comment (SteveL-MSFT) [inline .github/ISSUE_TEMPLATE.md]: Makes sense

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by iSazonov** [comment]
LGTM.

**Pull request merged**